### ISSUE TITLE
preview doesn't show value from mysite lang files

### ISSUE THREAD

**Issue by guyvanbael** [tractorcow-farm/silverstripe-fluent#470]
The page shows the exact values form the lang yml files. The preview in the cms however doesn't
<img width="1897" alt="36540604-567792ae-17db-11e8-8d79-1679ae93906f" src="https://user-images.githubusercontent.com/3098809/45865404-b03ab800-bd7d-11e8-962d-2aff12b2acbe.png">
<img width="1910" alt="36540605-568f185c-17db-11e8-87d7-f6e142821549" src="https://user-images.githubusercontent.com/3098809/45865405-b0d34e80-bd7d-11e8-8143-0922c2ce324b.png">

**Comment by GuySartorelli** [comment]
For what it's worth, the CMSPreview is explicitly _not_ considered "front end" in InitStateMiddleware::getIsFrontend
```php
// If using the CMS preview, do not treat the site as frontend
if ($request->getVar('CMSPreview')) {
    return false;
}
```

**Comment by tractorcow** [comment]
Right, but even if it's not "front end" it should still be setting the `i18n::set_locale` correctly, which is what would trigger the yml lang files to be respected.

Sorry I just realised this bug is from years ago... I need to check if this still is valid in fluent 5.
